pkg/standalone/standalone.go

```diff
@@ -1055,6 +1055,7 @@ func downloadFile(dir string, url string) (string, error) {
 			}).Dial,
 			TLSHandshakeTimeout:   15 * time.Second,
 			ResponseHeaderTimeout: 15 * time.Second,
+			Proxy: http.ProxyFromEnvironment,
```

Comment:
Here it is not correctly indented thats why the lint failure. Run `gofmt -w <filepath>` to correct this.

Reply:
Thank you for your correct.  :>